extensions/eclipse-che-theia-about/src/browser/style/che-theia-about.css

```diff
@@ -29,6 +29,7 @@
 .che-theia-about-product-logo > img {
     max-height: 100px;
     margin-right: 10px;
+    width: 200px;
```

Comment:
I'm afraid it will stretch the image. Could we avoid to use this property?

Reply:
will check

Reply:
Replaced with `height: 80px`.
It's the only way have both variants working: default logos and provided through a json file.